code/modules/mob/living/living_status_procs.dm

```diff
@@ -845,6 +845,12 @@ STATUS EFFECTS
 		singlemutcheck(src, block, MUTCHK_FORCED)
 		dna.UpdateSE()
 
+// Revivability
+
+
+/mob/living/proc/IsRevivable()
```

Comment:
```suggestion
/mob/living/proc/is_revivable()
```

Reply:
I was mostly following the convention of the existing status effect procs

Reply:
Let's break that convention and go with snake case 😄